script/plugin_webpack_directories.rb

```diff
@@ -25,11 +26,11 @@
   end
   path = "#{dep.to_spec.full_gem_path}/webpack"
   entry = "#{path}/index.js"
-  # some plugings share the same base directory (tasks-core and tasks, REX etc)
+  # some plugins share the same base directory (tasks-core and tasks, REX etc)
   # skip the plugin if its path is already included
   next if config[:paths].include?(path)
   if File.exist?(entry)
-    bundle_name = dep.name.gsub(/-|_|#{PLUGIN_NAME_REGEXP}/,'')
+    bundle_name = dep.name.gsub(/-|_|#{REMOVE_BUNDLE_NAME_PLUGINS}/,'')
```

Comment:
The real question here is should we be doing this even for foreman?  I would argue that the bundle name of foreman should probably be 'foreman' and not ''.

Reply:
@ohadlevy thoughts on this?

Reply:
In other words I would prefer to just remove this line.  I think it makes sense for foreman to reference its own bundle as 'foreman' just like everything else.

Reply:
@waldenraines do you mind if we just merge as is to unblock katello? changing the bundle name might have other impact on webpack configuration etc which i think we can look into in another pr?